Sources/Bookmarks/MenuBookmarksViewModel.swift

```diff
@@ -58,16 +60,25 @@ public class MenuBookmarksViewModel: MenuBookmarksInteracting {
         registerForChanges()
     }
 
+    deinit {
+        if let observer {
+            NotificationCenter.default.removeObserver(observer)
+            self.observer = nil
+        }
+    }
+
     private func registerForChanges() {
-        NotificationCenter.default.addObserver(forName: NSManagedObjectContext.didSaveObjectsNotification,
-                                               object: nil,
-                                               queue: .main) { [weak self] notification in
+        observer = NotificationCenter.default.addObserver(forName: NSManagedObjectContext.didSaveObjectsNotification,
```

Comment:
Why not use the selector version now?  And then you also don't need all this extra code for unregistering, right?

https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver

Reply:
I was thinking about that, and leaned towards not doing so:
- I don't like having to mark function as @objc just to use that API :)
- This shows good example of having to remove observer when using this queue based API - so I though about keeping that in the codebase, as it is easy to forget that some of the Notification Center API does handle unregistering on its own, and some not.

I don't have strong opinion tho, so if you care/feel it should be changed I can do this.

Reply:
Nah, I don't feel that strongly either :)